src/lib/structures/managers/ModerationManager.ts

```diff
@@ -47,16 +47,16 @@ export class ModerationManager extends Collection<number, ModerationEntity> {
 	/**
 	 * The channel where messages have to be sent.
 	 */
-	public get channel() {
-		const channelID = this.guild.settings.get(GuildSettings.Channels.ModerationLogs);
-		return (channelID && (this.guild.channels.cache.get(channelID) as TextChannel)) || null;
+	public async fetchChannel() {
+		const channelID = await this.guild.readSettings(GuildSettings.Channels.ModerationLogs);
+		return channelID ? ((this.guild.channels.cache.get(channelID) ?? null) as TextChannel | null) : null;
```

Comment:
This is an unreadable mess... 
```suggestion
		return this.guild.channels.cache.get(channelID) ?? null;
```

Reply:
![image](https://user-images.githubusercontent.com/24852502/98568663-a5bb3f80-22b1-11eb-9a18-0ba7749cb9ea.png)